### DISCUSSION TITLE
有没有具体配置表情包的方法

### DISCUSSION THREAD

**Discussion by w0oo** [walinejs/waline#1073] [Q&A] [answered]
![Screenshot_2022-05-29-10-22-23-11_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/105968998/170849361-68208583-4b46-4c43-aa82-0c22fa5e9bd3.jpg)
默认没有，

https://waline.js.org/guide/client/emoji.html#%E6%A0%B7%E5%BC%8F%E9%97%AE%E9%A2%98

这个链接，

**Comment by lizheming** [comment, marked as answer]
这个不需要配置默认是开启使用的，官网可能需要多刷新几次更新下缓存

**Reply by w0oo** [reply]
Gif用的是jsd还是unpkg.com？

**Reply by lizheming** [reply]
和 emoji 表情包不同，这是接入了第三方平台的 API，所以是用的对方的服务，没有用你说的这两个

**Reply by w0oo** [reply]
> 和 emoji 表情包不同，这是接入了第三方平台的 API，所以是用的对方的服务，没有用你说的这两个

好的

**Comment by w0oo** [comment]
az

**Comment by z7sz** [comment]
我一直搜索不出来

**Reply by lizheming** [reply]
可能被墙了，如果服务不可用可以隐藏掉。

**Reply by z7sz** [reply]
确实是被墙了，翻墙就可以，所以这个功能就跟没有一样，怎么隐藏掉？

**Reply by JarHMJ** [reply]
> 确实是被墙了，翻墙就可以，所以这个功能就跟没有一样，怎么隐藏掉？

可以通过自定义css的方式隐藏

```css
button.wl-action[title="表情包"] {
    display: none;
}
```
